#
# Detecting features:

manual_selection: true

# Multiplier for the max/min eigenvalue; specifies the minimal accepted quality of image corners
quality_level: 0.05
# Limit, specifying the minimum possible distance between the returned corners; Euclidian distance is used
minimum_dist: 5.0

#
# Tracking features:

# Size of the search window at each pyramid level
windows_size: 15
# 0-based maximal pyramid level number. If 0, pyramids are not used (single level), if 1, two levels are used etc.
max_level: 5
# Number of iterations before to stop
max_count: 20
# Minimum accuracy required to stop
epsilon: 0.3
#The relative weight of the spatial image derivatives impact to the optical flow estimation. If derivLambda=0 , only the image intensity is used, if derivLambda=1 , only derivatives are used. Any other values between 0 and 1 means that both derivatives and the image intensity are used (in the corresponding proportions).
deriv_lambda: 0
